universal-login-sdk/lib/integration/ethereum/BalanceChecker.ts

```diff
@@ -0,0 +1,23 @@
+import {providers, Contract} from 'ethers';
+import IERC20 from 'openzeppelin-solidity/build/contracts/IERC20.json';
+import {ETHER_NATIVE_TOKEN} from '@universal-login/commons';
+
+export class BalanceChecker {
+  constructor(private provider: providers.Provider, private walletAddress: string) {}
+
+  getBalance(tokenAddress: string) {
```

Comment:
code duplication? [balance.ts](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-commons/lib/integration/ethereum/balance.ts)

Reply:
Yes, we should deduplicate in the same PR.